assets/translations/en-us.yaml

```diff
@@ -1309,7 +1329,7 @@ landing:
     forums: Forums
   commercial:
     title: Commercial Support
-    body: Learn about commercial support <a href="https://rancher.com/support-maintenance-terms/" target='_blank' rel='noopener nofollow noreferrer'>here</a>.
+    body: Learn about commercial support <a href="/support">here</a>.
```

Comment:
This needs to be done as a nuxt link - this causes a full UI reload when clicked.

Reply:
Done